projects/samples/environments/indoor/worlds/break_room.wbt

```diff
@@ -0,0 +1,1190 @@
+#VRML_SIM R2020a utf8
+WorldInfo {
+  info [
+    "Copy of a real break room layout."
+    "This world contains the domestic objects included in Webots."
+    "Doors are interactive."
+    "This means that a robot could open a drawer, leave an object inside it, and close it."
+  ]
+  title "Break room layout"
+}
+Viewpoint {
+  orientation 0.5611620507608208 0.6084366364002342 0.5611613068199546 4.234793484858312
+  position -2.636688508521433 21.99638721155897 -0.12550161240670465
```

Comment:
```suggestion
  orientation 0.5278451084358184 0.8291023496586816 0.18430636258912056 5.404163177844073
  position -9.004955229768372 7.6584422165727934 10.196268121687902
```
The top view is not the best IMO because it doesn't showcase well the 3D of Webots.